apps/web-portal/lib/middlewares.js

```diff
@@ -0,0 +1,24 @@
+import cors from 'cors'
+import bodyParser from 'body-parser'
+
+export default {
+  json: bodyParser.json({
+    limit: 1024 * 1024,
+    verify: (req, res, buf) => {
+      try {
+        JSON.parse(buf)
+      } catch (e) {
+        res.send({
+          status: 0,
+          error: {
+            code: 'BROKEN_JSON',
+            message: 'Please, verify your json',
+          },
+        })
+        throw new Error('BROKEN_JSON')
+      }
+    },
+  }),
+  urlencoded: bodyParser.urlencoded({extended: true}),
+  cors: cors({origin: '*'}), // Allow any origin because NOT USE cookies and basic auth
```

Comment:
Add a TODO here to configure cors according to auth,

also, make an origin configurable with app config

Reply:
done